### ISSUE TITLE
URL in auth_functions are relative 

### ISSUE THREAD

**Issue by ARGO1960** [msaad1999/PHP-Login-System#25]
Hi, 
Great base to work with thanks for your work!!.

I noticed that in "/assets/includes/auth_functions.php" the header("Location: ../home/"); are relative.
that means they can only be included from an level one dir.
maybe header("Location: /home/"); is an solution or an entry in the env.php file.. that way its only one change...

**Comment by msaad1999** [comment]
Good idea, I do like having as many shared constants as possible present in env files, that ensures consistency in the app and makes changing anything easier. I will make this change along with some more as soon as I get some time.
